src/main/java/com/deque/html/axecore/selenium/HtmlReporter.java

```diff
@@ -0,0 +1,340 @@
+/*
+ * Copyright (C) 2020 Deque Systems Inc.,
+ *
+ * Your use of this Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ *
+ * This entire copyright notice must appear in every copy of this file you
+ * distribute or in any file that contains substantial portions of this source
+ * code.
+ */
+
+package com.deque.html.axecore.selenium;
+
+import com.deque.html.axecore.results.Node;
+import com.deque.html.axecore.results.Results;
+import com.deque.html.axecore.results.Rule;
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Collections;
+import java.util.Date;
+import java.util.HashSet;
+import java.util.List;
+
+import org.apache.commons.io.FileUtils;
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+import org.jsoup.nodes.Element;
+import org.openqa.selenium.OutputType;
+import org.openqa.selenium.SearchContext;
+import org.openqa.selenium.TakesScreenshot;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.WrapsElement;
+
+public class HtmlReporter {
+
+  private static final AxeBuilder axeBuilder = new AxeBuilder();
+
+  protected static final ResultType[] all = new ResultType[]
+      { ResultType.Passes, ResultType.Violations, ResultType.Incomplete, ResultType.Inapplicable};
+
+  private HtmlReporter() {
+  }
+
+  public static void createAxeHtmlReport(WebDriver webDriver, String destination)
+      throws IOException, ParseException {
+    createAxeHtmlReport(webDriver, destination, all);
+  }
+
+  public static void createAxeHtmlReport(WebDriver webDriver, String destination, ResultType[] requestedResults)
```

Comment:
not sure if to use an Array or a list. Using a list will allow the deletion of Report.java

Reply:
changed to using lists